srcpkgs/avidemux/patches/qt5.15.patch

```diff
@@ -0,0 +1,24 @@
+From 1a8bb36c5a0392d1e4d1b23812a926dde2796d5c Mon Sep 17 00:00:00 2001
+From: eumagga0x2a <eumagga0x2a@users.noreply.github.com>
+Date: Fri, 5 Jun 2020 11:41:37 +0200
+Subject: [PATCH] [Qt] Fix build with Qt 5.15
+
+---
+ avidemux/qt4/ADM_UIs/src/T_RubberControl.cpp | 4 ++++
+ 1 file changed, 4 insertions(+)
+
+diff --git avidemux/qt4/ADM_UIs/src/T_RubberControl.cpp avidemux/qt4/ADM_UIs/src/T_RubberControl.cpp
+index 59180d1dd..3c115a824 100644
+--- avidemux/qt4/ADM_UIs/src/T_RubberControl.cpp
++++ avidemux/qt4/ADM_UIs/src/T_RubberControl.cpp
+@@ -12,6 +12,10 @@
+ #include <QBoxLayout>
+ #include <QSizeGrip>
+ 
++#if QT_VERSION >= QT_VERSION_CHECK(5,15,0)
```

Comment:
I suppose the patch is from upstream? Since that `#if` is totally unnecessary.

Reply:
Yeah, it's a cherry pick from upstream -- I just needed it for local to build.